app/views/job_invocations/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<h2><%= @job_invocation.job_name %></h2>
+
+Template invocations:
```

Comment:
This string won't be here, no user would understand "Template invocation" term, the whole page is just debugging output for now.